restore/data.go

```diff
@@ -115,21 +114,14 @@ func restoreSingleTableData(queryContext context.Context, fpInfo *filepath.FileP
 		if copyErr != nil {
 			gplog.Error(copyErr.Error())
 			if MustGetFlagBool(options.ON_ERROR_CONTINUE) {
-				if connectionPool.Version.AtLeast("6") && backupConfig.SingleDataFile {
+				if connectionPool.Version.AtLeast("6") && (backupConfig.SingleDataFile || resizeCluster) {
```

Comment:
what if we encounter the error during resize from smaller to bigger segment number? The skip files should be created anyway?

Reply:
may be add check for bacthes and if it is 1 then don't create skip file?

Reply:
maybe just `batches > 0`? 
Again, is it valid behaviour to not creating skip files?

Reply:
`batches` is count of batches

Reply:
> is it valid behaviour not creating skip files?

I think no: in case if table does not exist we create skip file and with resize from smaller to lager and helper skips relation on first loop

Reply:
Yep, you are totaly correct. And if this number is greater than 1 we have several batches for one tableoid, which we want to skip and we can create skip files. If we have single batch `batches=1` we don't need the skip file. In case of restoring on bigger number of segments `batches=1` as well.
Again, is it valid not to create skip files in case of single batch?

Reply:
> is it valid not to create skip files in case of single batch?

in case if table does not exist we create skip file and helper skips relation on first loop